### ISSUE TITLE
ColorPicker is not working in linux mint 17

### ISSUE THREAD

**Issue by barkbaek** [weslly/ColorPicker#75]
I'm using Sublime Text 3 in linux mint 17.
I overrided default keybindings on [Preferences] - [Keybindings - User] as
_________________________________________________________
[
    { "keys": ["ctrl+shift+c"], "command": "color_pick" }
]
_________________________________________________________

I tried to run Color Picker in Command Palette and also tried to run it by shortcuts("ctrl+shift+c"),
but it is not working.
I have tried to figure it out for 5 hours..
How can I solve this problem ?

**Comment by weslly** [comment]
Does it show any error message in the console when you use the shortcut? (Use <kbd>ctrl</kbd>+<kbd>`</kbd> to open the console).

**Comment by barkbaek** [comment]
No, nothing is shown in the console when I use the shortcut or Command Palette.
I have a question about ColorPicker's path.
Where should it be installed ?
My path of ColorPicker is "sublime-text-3/Packages/ColorPicker".
But other packages(e.x. ) are installed both in "sublime-text-3/Installed Packages/" and "sublime-text-3/Packages/User/".
Is it the right path? 
Please help me I really want to use the great feature of Color Picker.

**Comment by weslly** [comment]
Have you tried to run `ColorPicker/lib/linux_colorpicker.py` from the terminal (outside sublime)?

**Comment by barkbaek** [comment]
Yes I've tried to run it and it's working well.

**Comment by weslly** [comment]
If it doesn't display any errors on the console and linux_colorpicker.py works outside sublime, I don't know what is going wrong with it. Did you try to reinstall it from package control?

**Comment by barkbaek** [comment]
Yes I tried to reinstall it about 10 times, but I cannot find the reason of this issue..

**Comment by weslly** [comment]
I'm not really sure about what it can be. Have you tried with a fresh install? Maybe there's a conflict with another plugin.

**Comment by charlesmudy** [comment]
I had similar problem and what I discovered was that another plugin on my OSX was using that shortcut which stops it from displaying. Changing the shortcut for that plugin solves it.

**Comment by mbo42** [comment]
Just FYI, It wouldn't work for me in a virtualenv as it depends on python-gtk, and I didn't want to install that.
It wouldn't hurt to print an error on the console if all the attempts at import gtk fail (linux_colorpicker.py)
